### DISCUSSION TITLE
Incorrect string interpolation in prepare-pyodide.js

### DISCUSSION THREAD

**Discussion by Eggze2** [open-webui/open-webui#5187] [General]
When I try to install and build from Open WebUI Github Repo using npm in Ubuntu22.04, the error occurs due to incorrect string interpolation in JavaScript. The paths for node_modules/pyodide/${entry} and static/pyodide/${entry} are not wrapped in backticks (```), which are required for template literals. This results in a syntax error.

Fix:
Replace single quotes or missing quotes with backticks to properly interpolate variables within strings.

Example correction:

```javascript
console.log(`Installing package: ${pkg}`);
await copyFile(`node_modules/pyodide/${entry}`, `static/pyodide/${entry}`);
```

This fixes the interpolation issue and allows the script to run as expected.

**Comment by justinh-rahb** [comment]
PR welcomed!
